stdlib/universe/elapsed.go

```diff
@@ -0,0 +1,234 @@
+package universe
+
+import (
+	"fmt"
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/execute"
+	"github.com/influxdata/flux/plan"
+	"github.com/influxdata/flux/semantic"
+	"time"
+)
+
+const ElapsedKind = "elapsed"
+
+type ElapsedOpSpec struct {
+	Unit       flux.Duration `json:"unit"`
+	TimeColumn string        `json:"timeColumn"`
+	ColumnName string        `json:"columnName"`
+}
+
+func init() {
+	elapsedSignature := flux.FunctionSignature(
+		map[string]semantic.PolyType{
+			"unit":       semantic.Duration,
+			"timeColumn": semantic.String,
+			"columnName": semantic.String,
+		},
+		nil,
+	)
+
+	flux.RegisterPackageValue("universe", ElapsedKind, flux.FunctionValue(ElapsedKind, createElapsedOpSpec, elapsedSignature))
+	flux.RegisterOpSpec(ElapsedKind, newElapsedOp)
+	plan.RegisterProcedureSpec(ElapsedKind, newElapsedProcedure, ElapsedKind)
+	execute.RegisterTransformation(ElapsedKind, createElapsedTransformation)
+}
+
+func createElapsedOpSpec(args flux.Arguments, a *flux.Administration) (flux.OperationSpec, error) {
+	if err := a.AddParentFromArgs(args); err != nil {
+		return nil, err
+	}
+
+	err := a.AddParentFromArgs(args)
```

Comment:
This looks like a duplicate from the line above. I think you can remove it.

Reply:
if i remove line 41, then `err` will never have been declared (b/c it was previously in an `if` block)